controllers/progressivesync_controller.go

```diff
@@ -143,7 +149,7 @@ func (r *ProgressiveSyncReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		// TODO: status status - update targets
 
 		for _, s := range scheduledApps {
-			r.Log.Info("syncing app", "app", s)
+			log.Info("syncing app", "app", s)
```

Comment:
Why is this changing?

Reply:
```
log := r.Log.WithValues("ProgressiveSync", req.NamespacedName)
```

I figured it would be more useful to log to which instance of progressivesync object do these logs pertain to.

Reply:
my bad, i missed this.